mobile/src/home-view.js

```diff
@@ -56,6 +57,25 @@ class HomeView extends Component {
         this.setState({ sessions: [...this.state.sessions, {...data.val(), key: data.key }] })
       })
 
+      sessRef.on('child_changed', data => {
+        var sessions = this.state.sessions
+        for (var i in sessions) {
+          if (sessions[i].key === data.key) {
+            sessions[i].sessionName = data.val().sessionName
+            if (this.state.session.key === data.key) {
+              var newSession = this.state.session
+              newSession.sessionName = data.val().sessionName
+              this.setState({sessions, session: newSession })
+              break;
```

Comment:
In both branches of this inner `if`, you `break;`, so just put the break after this inner `if`

Reply:
updated